src/components/Disclaimer.tsx

```diff
@@ -0,0 +1,43 @@
+/**
+ * Copyright (c) 2025-present, NDLA.
+ *
+ * This source code is licensed under the GPLv3 license found in the
+ * LICENSE file in the root directory of this source tree.
+ *
+ */
+
+import { useMemo } from "react";
+import { useTranslation } from "react-i18next";
+import { transform } from "@ndla/article-converter";
+import { AccessibilityFill } from "@ndla/icons";
+import { Button, PopoverContent, PopoverRoot, PopoverTitle, PopoverTrigger } from "@ndla/primitives";
+import { GQLTransformedDisclaimerContent } from "../graphqlTypes";
+
+interface Props {
+  disclaimer: GQLTransformedDisclaimerContent;
+}
+
+const Disclaimer = ({ disclaimer }: Props) => {
+  const { t } = useTranslation();
+
+  const transformedDisclaimer = useMemo(() => {
+    return transform(disclaimer.content, {});
```

Comment:
Sende inn opts?

Reply:
Vil vi egentlig det? hvilke tenkte du på isåfall?

Reply:
Subject brukes vel for å transformere lenker, gjør det ikke?

Reply:
Einaste embed som kan være i disclaimer er content-link og der brukes ikkje subject.